src/app/app-routing.module.ts

```diff
@@ -1,10 +1,22 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { SearchRouteComponent } from './views/search-route/search-route.component';
-import {NavBarComponent} from './components/nav-bar/nav-bar.component';
+import { NavBarComponent } from './components/nav-bar/nav-bar.component';
+import { PreferencesComponent } from './views/preferences/preferences.component';
+import { AboutComponent } from './views/about/about.component';
+import { CustomVehiclesComponent } from './views/preferences/custom-vehicles/custom-vehicles.component';
+import { Co2GoalsComponent } from './views/preferences/co2-goals/co2-goals.component';
+import { ManageDataComponent } from './views/preferences/manage-data/manage-data.component';
+import { EditVehicleComponent } from './views/preferences/custom-vehicles/edit-vehicle/edit-vehicle.component';
```

Comment:
I really think we should sort imports by imported component names, shouldn't we?

Reply:
277566b